tests/units/clean/test_clean.py

```diff
@@ -0,0 +1,39 @@
+
+import os
+import logging
+import subprocess
+import datetime
+from pytest import fixture
+from unittest import mock
+from freezegun import freeze_time
+
+from clean.clean import cleanup_command
+
+
+@fixture
+@freeze_time("2023-10-20")
+def create_log():
+    logs_date = datetime.datetime.now()
+    log_path = f'tests/units/clean/logs/dag_id=test/run_id=test__{logs_date.strftime("%Y-%m-%dT%H:%M:%S.%f%z")}/task_id=test_task/attempt=1.log'
+    os.makedirs(os.path.dirname(log_path), exist_ok=True)
+    try:
+        with open(log_path, "w") as f:
+            f.write("test log")
+        return True
+    except:
+        logging.error('Failed file creation for tests')
+        pass
```

Comment:
I would recommend not to  write on the file system in the tests, you could use virtual temp files in python to test the same functionality :)